Sources/TuistAnalytics/CommandEvent.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+public struct CommandEvent: Codable {
```

Comment:
I know you've discarded the idea for now, but I still feel that having additional details like: `number of target used in a tuist focus command` would be very interesting. Not a deal breaker for me but I thought I'd share that!

Reply:
In this spirit, I'd add something like this in this struct:

```swift
let properties: [String: String]
```

Reply:
Done